src/components/svg/LinkSvg.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { ISvgProps, Svg } from './Svg';
+import * as classname from 'classnames';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+
+export interface ILink {
+  url: string;
+  target?: string;
+}
```

Comment:
Is it usefull to have an object in the LinkSVGProps instead of two properties? The default is probably not deep (it won't set the target if you pass `{link: {url: 'testurl'}}` )

Reply:
I will add this 2 props in the ILinkSvgProps